GoTwitterCloneBackend/GoTwitterCloneApp/controller/followhandlers/startfollowhandler.go

```diff
@@ -28,14 +29,7 @@ func StartFollowHandler(w http.ResponseWriter, r *http.Request) {
 		Username:         targetFollowUsername,
 		FollowerUsername: requesterUsername,
 	}
-
-	err = followservice.StartFollowing(r.Context(), startFollowRequest)
-	if err != nil {
-		responseHelper.SetJsonResponse(http.StatusBadRequest, []string{
-			"start_follow_attempt_failed",
-		})
-		return
-	}
+	events.PublishEventMessage(r.Context(), eventhandlers.FollowEventHandlerName, startFollowRequest)
```

Comment:
trigger follow event handler to separate execution from http request. should reduce latency